src/_sass/_search.scss

```diff
@@ -0,0 +1,58 @@
+@use "./colours";
+
+.search-form {
+  display: flex;
+
+  &__input {
+    background-color: colours.$ivory;
+    border: 0.063rem solid colours.$navy;
+    color: colours.$navy;
+    line-height: 1.35;
+    margin-right: -0.063rem;
+    padding: 0.5rem 0.938rem;
+    width: 100%;
+  }
+
+  &__submit {
+    background-color: colours.$yellow-light;
+    border: 0.063rem solid colours.$navy;
```

Comment:
The navbar background changed midway through development and is now the same as this border. It's still visible between the text input and the submit button, but not elsewhere. Maybe we should simplify to `border-right`/`border-left`, or pick another colour (I think it's probably fine without one on the outsides)